crates/nu-command/src/debug/info.rs

```diff
@@ -146,8 +146,8 @@ fn all_columns(span: Span) -> Value {
                 "root" => root,
                 "cwd" => cwd,
                 "exe_path" => exe_path,
-                "command" => Value::string(p.cmd().join(" "), span),
-                "name" => Value::string(p.name(), span),
+                "command" => Value::string(p.cmd().join(std::ffi::OsStr::new(" ")).to_string_lossy(), span),
```

Comment:
Not really a problem of this PR but is this is probably a dangerous lie with potential quoting rules on the different platforms?

Reply:
Yeah `debug info` probably get's less use from the general public than something like `ps`